internal/namespaces/instance/v1/custom_privatenics.go

```diff
@@ -31,3 +33,27 @@ func privateNicListBuilder(c *core.Command) *core.Command {
 
 	return c
 }
+
+func privateNicCreateBuilder(c *core.Command) *core.Command {
+	c.Interceptor = func(ctx context.Context, argsI interface{}, runner core.CommandRunner) (i interface{}, err error) {
+		res, err := runner(ctx, argsI)
+		if err == nil {
+			return res, nil
+		}
+
+		resErr, isResErr := err.(*scw.InvalidArgumentsError)
+		if !isResErr {
+			return nil, err
+		}
+
+		if resErr.Details[0].ArgumentName == "private_network_id" && resErr.Details[0].HelpMessage == "required key not provided" {
```

Comment:
Why is this not handled by command generation? Could we mark an argument as required and let the generation handle it?